src/main/java/seedu/address/MainApp.java

```diff
@@ -82,27 +83,36 @@ public void init() throws Exception {
     private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         Optional<ReadOnlyAddressBook> addressBookOptional;
         ReadOnlyAddressBook initialData;
-        RoomList readOnlyRoomOccupancy;
+        Optional<ReadOnlyRoomList> readOnlyRoomOccupancy;
+        ReadOnlyRoomList initialRoomList;
+
         try {
             readOnlyRoomOccupancy = storage.readRoomOccupancyStorage();
-        } catch (IOException ioe) {
-            readOnlyRoomOccupancy = new RoomList();
+            initialRoomList = readOnlyRoomOccupancy.orElseGet(SampleDataUtil::getSampleRoomList);
+        } catch (DataConversionException e) {
+            logger.warning(
+                    "Room Data file not in the correct format. Will be starting with an empty AddressBook");
```

Comment:
Will be starting with an empty "room list" or something instead of AddressBook. Just to clean up traces of address book from the code haha.

Reply:
That was wat I was wondering too, but didn't want to delete before discussing.